src/resources/locales/en.json

```diff
@@ -854,9 +854,229 @@
             "disclaimer": "Shortcuts were defined from QWERTY keyboard layout.",
             "editUserJson": "Edit user file...",
             "keyboardShortcuts": "Keyboard shortcuts",
+            "list": {
+                "AddBookmarkWithLabel": {
+                    "description": "",
+                    "name": "Add Bookmark with Label"
+                },
+                "AnnotationsCreate": {
+                    "description": "When a text is selected, it will be marked as annotation and the annotation dialog box will open to allow additional text and select of color and stroke.",
```

Comment:
Perhaps we need to improve this long description for the `annotationCreate`, what does it means : "It will be marked as annotation` !?

Reply:
The annotation (aka Note) will be saved after annotation popup submitted